lib/enivronmentalist.ex

```diff
@@ -4,10 +4,13 @@ defmodule Enivronmentalist do
   end
 
   def process([]) do
-    sampe_file = EnvFileParser.for_file(".env")
   end
 
   def process(options) do
+    sample_file = EnvFileHandler.parse(options[:sample_file])
+    actual_file = EnvFileHandler.parse(options[:actual_file])
```

Comment:
Why use underscores for the options?  I feel this would make typing this out via command line rather lame.  Avoiding hitting the shift key is minor but a nice convenience.